src/RightsService/Controllers/RolesController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using LT.DigitalOffice.RightsService.Business.Role.Interfaces;
+using LT.DigitalOffice.RightsService.Models.Dto;
+using LT.DigitalOffice.RightsService.Models.Dto.Responses;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.RightsService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class RolesController : ControllerBase
+    {
+        [HttpGet("find")]
+        public RolesResponse Find(
+            [FromServices] IFindRolesCommand command,
+            [FromQuery] int skipCount,
+            [FromQuery] int takeCount)
+        {
+            return command.Execute(skipCount, takeCount);
+        }
+
+        [HttpPost("create")]
+        public void Create(
+            [FromServices] ICreateRoleCommand command,
+            [FromQuery] CreateRoleRequest role)
+        {
+            command.Execute(role);
```

Comment:
When we create smth we have to return status code Created (201)

Reply:
done